.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please delete this file as it belongs to vs code. See here: https://docs.microsoft.com/en-us/azure/devops/repos/git/ignore-files?view=azure-devops&tabs=visual-studio-2019#use-a-global-gitignore-file

Reply:
with new commit, issue should be solved